apps/delivery-options/src/composables/useFeatures.ts

```diff
@@ -1,19 +1,25 @@
 import {computed} from 'vue';
 import {useMemoize} from '@vueuse/core';
-import {ConfigSetting, PACKAGE_TYPE_DEFAULT} from '@myparcel-do/shared';
+import {ConfigSetting, LimitedPackageTypeName} from '@myparcel-do/shared';
 import {useConfigStore} from '../stores';
 import {SHOWN_SHIPMENT_OPTIONS} from '../data';
 
+const PACKAGE_TYPE_DEFAULT = [LimitedPackageTypeName.Package, LimitedPackageTypeName.PackageSmall];
+
 export const useFeatures = useMemoize(() => {
   const config = useConfigStore();
-
   return {
     availableShipmentOptions: computed(() => {
-      return PACKAGE_TYPE_DEFAULT === config.packageType ? SHOWN_SHIPMENT_OPTIONS : [];
+      return PACKAGE_TYPE_DEFAULT.includes(config.packageType as unknown as LimitedPackageTypeName)
+        ? SHOWN_SHIPMENT_OPTIONS
+        : [];
```

Comment:
If you put this back on one line, your coverage improves probably.

Reply:
That is a good suggestion, Ill revert this, it was a linting fix